wireapp/wire-android
# ci: remove usage of deprecated ::set-output (AR-3375)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-3375" title="AR-3375" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />AR-3375</a>  Github Workflow uses outdated `set-output` functionality
  </td></table>
  <br />
 




----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues
`::set-output` will be deprecated on 2023-05-31 (
see https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/ for more details)

### Solutions

upgrade to new syntax as described in the blog post


### Testing

This was not tested, final proof that it works can only be shown when the timeline has run.

#### How to Test

Please verify if the change looks as suggested by the documentation/blog post.


----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
